.gitignore

```diff
@@ -149,3 +149,49 @@ test/data/*.json
 test/data/pipeline/*.json
 test/data/filters/*.json
 test/data/plang/*.json
+
```

Comment:
I'd probably prefer not polluting this PR with `.gitignore` changes. If you think the project's `.gitignore` should be updated, that should be done in another PR. You can use a local `.gitignore` to ignore files that relate to you and your editors (e.g. I ignore `build/` locally).